mujoco_simulator:
  # Path to your MJCF (.xml) model file.
  # Example: "/home/ubuntu/ros2_ws/src/eiriarm_description/mjcf/eiriarm.xml"
  modelFile: "/home/ubuntu/ros2_ws/src/description/dual_arm_support/mjcf/dual_arm_robot.xml"
  jointCommandsTopic: "/ctrl/effort"
  unPauseService: "/unpause_physics"
  initPauseFlag: true
  modelTableFlag: true
